newdle/client/src/components/creation/timeslots/TimeslotsStep.js

```diff
@@ -41,13 +41,17 @@ export default function TimeslotsStep() {
     <div>
       <Grid container>
         <Grid.Row columns={2}>
-          <Grid.Column width={5}>
-            <MonthCalendar />
-            <SelectedDates />
-          </Grid.Column>
-          <Grid.Column width={11}>
-            <Availability />
-          </Grid.Column>
+          <Grid stackable>
+            <Grid.Row columns={2}>
```

Comment:
I believe we could simplify this to:
```html
<Grid.Row>
    <Grid stackable>
...
```

Reply:
yes and now, as row adds to the presentation too 
![image](https://user-images.githubusercontent.com/9433061/93595991-fa7cc100-f9b8-11ea-88e9-c70ead78a252.png)

Reply:
I see what you mean. But is there any difference besides the minor padding?
As it looks a bit weird to duplicate a Row with two columns inside another Row with two columns.